db_stress_tool/db_stress_test_base.cc

```diff
@@ -1378,16 +1384,76 @@ void StressTest::OperateDb(ThreadState* thread) {
       } else if (prob_op < write_bound) {
         assert(prefix_bound <= prob_op);
         // OPERATION write
+        if (disable_fault_injection_during_put_delete) {
+          fault_fs_guard->DisableThreadLocalErrorInjection(
```

Comment:
These enable/disable code are pretty common, maybe move to a helper function.

Reply:
> Would it work if we just keep the expected state at pending state? I

Yeah but that will require too much change on the ExpectedState that needs some time to resonate so I lean toward not doing so.  I will address the rest of the TODO

Reply:
Fixed refactory and expected state